services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/settings.py

```diff
@@ -23,6 +23,10 @@ class DynamicSidecarSettings(BaseCustomSettings, MixinLoggingSettings):
         description="boot mode helps determine if in development mode or normal operation",
     )
 
+    DYNAMIC_SIDECAR_DY_VOLUMES_MOUNT_DIR: Path = Field(
+        ..., env=["DY_VOLUMES", "DYNAMIC_SIDECAR_DY_VOLUMES_MOUNT_DIR"]
```

Comment:
I would not keep the legacy version of `DY_VOLUMES` if you decided to replace it. Just use `DYNAMIC_SIDECAR_DY_VOLUMES_MOUNT_DIR` everywhere. There is no actual risk in replacing it in the codebase since it is used to share information between director-v2 and the dy-sidecar.